Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -955,7 +960,7 @@ void connectSimpleBusMessageCallback(GstElement* pipeline, Function<void(GstMess
             // This can happen if the latency of live elements changes, or
             // for one reason or another a new live element is added or
             // removed from the pipeline.
-            gst_bin_recalculate_latency(GST_BIN_CAST(pipeline.get()));
+            gst_element_call_async (pipeline.get(), recalculate_latency_async, NULL, NULL);
```

Comment:
Was this also triggering deadlocks? 

BTW s/NULL/nullptr in the whole patch please :)

Reply:
> Was this also triggering deadlocks?

Yes, same as the first one: main thread being blocked.